services/test_results.py

```diff
@@ -172,6 +172,20 @@ async def notify(
             )
             return False, "torngit_error"
 
+    def format_single_test(self, test_name, test_env_list):
+        testsuite_section = f"Testsuite:<br>{test_name.split('//////////')[0]}"
```

Comment:
is `'//////////'` something that we put there? Seems very random to have this many '/' in a row...

Also [nit] maybe create a variable for this value, especially if it's something we put there. It could come with a comment explaining why this exists...

Reply:
We did put it there, it's because I was trying to deduplicate failure messages, I wanted to have multiple tests that map to the same failure message in the same row, so I had to use a dict, and for that I had to put the testsuite and testname together as the key in the dict, but I needed a way to separate them back out so I joined using `/////////` when placing them as the key in the dict and then split it back out here.

I'm planning on removing this in my next change to the notifier.